src/__swaps__/screens/Swap/components/AnimatedChainImage.tsx

```diff
@@ -1,127 +1,7 @@
-import React from 'react';
```

Comment:
This file just tells each platform to import the corresponding file making typescript happy